docs/source/05_solver_config_hooks.rst

```diff
@@ -0,0 +1,60 @@
+.. _solver_config_hooks-section:
+
+Solver Configuration Hooks
+==========================
+
+The solver configuration hooks allow plugins configure internal settings from the ALFAsim.
+To add an configuration hook is necessary to implement the given `hook` in a python file that is already available
+on your plugin project folder.
+
+As an example, if the ``alfasim-sdk template`` command was created with the name ``myplugin`` the necessary file to
+be customized will be located on: :menuselection:`myplugin --> src --> python --> myplugin.py`
+
+.. contents::
+    :depth: 3
+    :local:
+
+Additional Variables
+--------------------
+
+.. autofunction:: alfasim_sdk.hook_specs_gui.alfasim_get_additional_variables
+
+.. _multi-field-description:
+
+Hydrodynamic Model
+-------------------
+
+|alfasim| provides a way to customize the hydrodynamic model available within the application, if the usage of
```

Comment:
I didn't get what you meant with "if the usage of the hook listed below"

Reply:
I think he wants to say " `with` the usage of the hook listed below " (typo if-> with)